### ISSUE TITLE
generation of preds_fed.h5 file

### ISSUE THREAD

**Issue by ShaminiKoravuna** [rayat137/Pose_3D#13]
Hello Rayat!

I have a few questions related to the generation of preds_fed.h5 file
1. Did you make any changes in the stacked_hourglass network model for generating the 2D pose estimation output? 
2. I'm unable to generate the predictions using the pytorch model. Do you have any tensorflow version of it?
3. If I have the files for each image in json format how can I convert them to .h5 format?

Thanks a lot for time & Support!

**Comment by rayat137** [comment]
Hi Shamini, 

Thanks for your interest in our work. Here are the responses to your queries: 

1. No I don't remember making any changes to the structure of stacked hourglass network. Although I did fine-tune on H3.6M dataset. 
2. I used the torch-lua code that the authors provided to generate the 2D predictions. PyTorch wasn't there at that time. 
3. So .h5 format saves everything as a dictionary of matrices. So you first need to read the Json file, and store the data as a numpy tensor.  Then when writing to the .h5 file give a name to that matrix. 

Hope this answers your concerns. Feel free to contact me if you have any questions.
